packages/react-styles/src/css/components/Topology/topology-pipelines.css

```diff
@@ -0,0 +1,362 @@
+  --pf-topology-pipelines__pill-background--Fill: var(--pf-topology-pipelines__pill-background--m-success--Fill);
+  --pf-topology-pipelines__pill-background--Stroke: var(--pf-topology-pipelines__pill-background--m-success--Stroke);
+}
+
+.pf-topology-pipelines__pill.pf-m-selected.pf-m-warning .pf-topology-pipelines__pill-background {
+  --pf-topology-pipelines__pill-background--Fill: var(--pf-topology-pipelines__pill-background--m-warning--Fill);
+  --pf-topology-pipelines__pill-background--Stroke: var(--pf-topology-pipelines__pill-background--m-warning--Stroke);
+}
+
+.pf-topology-pipelines__pill.pf-m-selected.pf-m-in-progress .pf-topology-pipelines__pill-background {
+  --pf-topology-pipelines__pill-background--Fill: var(--pf-topology-pipelines__pill-background--m-in-progress--Fill);
+  --pf-topology-pipelines__pill-background--Stroke: var(--pf-topology-pipelines__pill-background--m-in-progress--Stroke);
+}
+
+.pf-topology-pipelines__pill.pf-m-selected.pf-m-pending .pf-topology-pipelines__pill-background {
+  --pf-topology-pipelines__pill-background--Fill: var(--pf-topology-pipelines__pill-background--m-selected--m-pending--Fill);
+  --pf-topology-pipelines__pill-background--Stroke: var(--pf-topology-pipelines__pill-background--m-pending--Stroke);
+}
+
+.pf-topology-pipelines__pill.pf-m-selected.pf-m-running .pf-topology-pipelines__pill-background {
+  --pf-topology-pipelines__pill-background--Fill: var(--pf-topology-pipelines__pill-background--m-selected--m-running--Fill);
+  --pf-topology-pipelines__pill-background--Stroke: var(--pf-topology-pipelines__pill-background--m-running--Stroke);
+}
+
+.pf-topology-pipelines__pill.pf-m-selected.pf-m-idle .pf-topology-pipelines__pill-background {
+  --pf-topology-pipelines__pill-background--Fill: var(--pf-topology-pipelines__pill-background--m-selected--m-idle--Fill);
+  --pf-topology-pipelines__pill-background--Stroke: var(--pf-topology-pipelines__pill-background--m-idle--Stroke);
+}
+
+.pf-topology-pipelines__pill.pf-m-hover.pf-m-danger .pf-topology-pipelines__pill-background {
+  --pf-topology-pipelines__pill-background--StrokeWidth: var(--pf-global--BorderWidth--md);
+}
+.pf-topology-pipelines__pill.pf-m-hover.pf-m-success .pf-topology-pipelines__pill-background {
+  --pf-topology-pipelines__pill-background--StrokeWidth: var(--pf-global--BorderWidth--md);
+}
+.pf-topology-pipelines__pill.pf-m-hover.pf-m-warning .pf-topology-pipelines__pill-background {
+  --pf-topology-pipelines__pill-background--StrokeWidth: var(--pf-global--BorderWidth--md);
+}
+.pf-topology-pipelines__pill.pf-m-hover.pf-m-in-progress .pf-topology-pipelines__pill-background {
+  --pf-topology-pipelines__pill-background--StrokeWidth: var(--pf-global--BorderWidth--md);
+}
+
+/* Node label */
+.pf-topology-pipelines__pill-text {
+  font-size: var(--pf-topology-pipelines__pill-text--FontSize);
+  font-family: var(--pf-topology-pipelines__pill-text--FontFamily);
+  fill: var(--pf-topology-pipelines__pill--Color);
```

Comment:
I think this should actually be `--pf-topology-pipelines__pill-text--Color` and above line 42, you can initialize it to`--pf-topology-pipelines__pill-text--Color: var(--pf-topology-pipelines__pill--Color)`. Then `--pf-topology-pipelines__pill-text--Color` can be reassigned in each of the modifiers below rather than setting fill directly.
Then line 96 for dark theme then may or may not be needed (I think it's probably good to keep), but currently I think it's not being referenced at all.
Caveat: I didn't try all this, it's just on inspection 🤪

Reply:
@jeff-phillips-18 Do you want to reassign the variable rather than setting fill directly in lines 213-243? Otherwise, this PR LGTM. ⭐